### ISSUE TITLE
Detect use_inline_resources in HWRPs

### ISSUE THREAD

**Issue by tas50** [chef/cookstyle#787]
### What category of cop is this?:

Please try to select as few as possible.

- [ ] Chef/Deprecations - A cop that alerts on a deprecation in the Chef Infra Client
- [ ] Chef/Correctness - A cop that alerts when incorrect coding behavior is being used
- [ ] Chef/Sharing - A cop that alerts to missing functionality necessary for sharing cookbooks with others
- [ ] Chef/Style - A cop that alerts to a style best practice
- [x] Chef/Modernize - A cop that alerts when a cookbook can be simplified or modernized with new functionality
- [ ] Chef/Effortless - A cop that alerts on code that must be resolved to move to the Effortless pattern

### Describe the new cop:
There's no need for use_inline_resources in HWRPs anymore

### What it would trigger on:

```ruby
use_inline_resources
```

### What it would autocorrect to if applicable

```ruby
```

**Comment by tas50** [comment]
False alarm. This is fixed already
